tests/Domain/Mail/MailManagerTest.php

```diff
@@ -51,23 +51,31 @@ protected function setUp(): void
         );
 
         $this->contacts = [
-            new Contact(
+            'grote.smurf@publiq.be' => new Contact(
                 Uuid::uuid4(),
                 Uuid::fromString(self::INTEGRATION_ID),
                 'grote.smurf@publiq.be',
                 ContactType::Technical,
                 'Grote',
                 'Smurf'
             ),
-            new Contact(
+            'grote.smurf@publiq.be2' => new Contact( // This email will be merged into the previous contact
```

Comment:
I think it is not merged, it will be not added to the list because the same email is already present in the list and this is not a functional contact. Also, no real merge would happen if you change the order.

Reply:
I guess the word "merge" was badly chosen. I changed it for the word "discarded", which is more in line with what I meant.
I also added another contact in the list to prove that the order does not matter, the code works like intended.